submissions/favourImade/css/style.css

```diff
@@ -123,6 +123,7 @@ section#home .serve .serve-row div p{
     color: grey;
 }
 
+/*
```

Comment:
Your style sheet should be broken into two. The one for landing.html while that of the vendor page separately. They should work concurrently not one commented for the other to work.